app/src/org/commcare/android/view/GraphView.java

```diff
@@ -405,6 +405,7 @@ private void configureSeries(SeriesData s, XYSeriesRenderer currentRenderer) {
             currentRenderer.setPointStyle(style);
             currentRenderer.setFillPoints(true);
             currentRenderer.setPointStrokeWidth(2);
+            mRenderer.setPointSize(6);

```

Comment:
That is an interesting idea. Not going to do it right now, will keep in mind when migrating graph libraries.